src/gui/executionlog.cpp

```diff
@@ -28,15 +28,21 @@
  * Contact : chris@qbittorrent.org
  */
 
+#include "executionlog.h"
+
+#include "guiiconprovider.h"
```

Comment:
Second point: system includes are usually not sensitive to the include order, while app includes might have implicit dependencies. Listing them closer to the start of the includes list helps to find such dependencies if any.